miio/viomivacuum.py

```diff
@@ -63,7 +65,7 @@ class ViomiMode(Enum):
     Vacuum = 0  # No Mop, Vacuum only
     VacuumAndMop = 1
     Mop = 2
-
+    Unknown = 4
```

Comment:
I think it would make more sense to add a log message(s) to the place(s) where the unknown mode is encountered, otherwise this looks good to go, thanks for the PR! :+1:

Reply:
Ping?

Reply:
I'm not sure I understand your comment, you mean a log message in the `mode`  method ?
like here: https://github.com/rytilahti/python-miio/blob/5a3f2f446ff12210ba0c5fe0ede5016f4ca7c01e/miio/viomivacuum.py#L145 ?

Reply:
I tried to say that it's more useful to simply log the unknown state on the call site, like done in https://github.com/rytilahti/python-miio/blob/5a3f2f446ff12210ba0c5fe0ede5016f4ca7c01e/miio/viomivacuum.py#L125 --  in case someone is seeing those warnings in their logs and knows what causes them, they can create a PR to fix it.

Reply:
If @fs79 doesn't answer, I can integrate this patch in this PR (https://github.com/rytilahti/python-miio/pull/808)

Reply:
Mode `4` is something like `CleanPosition`.